package.json

```diff
@@ -6,8 +6,8 @@
   "type": "module",
   "scripts": {
     "dev": "parcel example/index.html",
-    "test": "mocha ./tests/**/*.test.js",
-    "test-watch": "mocha --watch --require @babel/polyfill --require @babel/register ./tests/**/*.test.js -r esm",
+    "test": "mocha --require ./tests/setupTests.js --require jsdom-global/register ./tests/**/*.test.js",
+    "test-watch": "mocha --watch --require ./tests/setupTests.js --require @babel/polyfill --require @babel/register --require esm ./tests/**/*.test.js",
```

Comment:
`test-watch` fails for me, unless I remove the requirements for babel. Should I go ahead and do that? I didnt end up using it much since its output isnt as useful as the `test` command.